clients/fides-js/src/fides-tcf.ts

```diff
@@ -128,6 +124,82 @@ const getInitialPreference = (
   return tcfObject.default_preference ?? UserConsentPreference.OPT_OUT;
 };
 
+/**
+ * Populates TCF entities with items from cookie.tcf_consent and Fides string.
+ * Returns TCF entities to be assigned to an experience.
```

Comment:
Our terminology is a bit confusing I realize. We talk about the cookie vs the Fides string, when here they are both already on the cookie at this point. Can we clarify in the comments that `TCF entities are derived from both cookie.tcf_consent and cookie.fides_string. We must look at both because they contain non-overlapping info that is required for a complete tcf entities object.  `

Reply:
updated